packages/@aws-cdk/core/lib/cfn-parse.ts

```diff
@@ -228,7 +231,7 @@ export interface ParseCfnOptions {
    * Values provided here will replace references to parameters in the parsed template.
    * @default - no parameters will be replaced
    */
-  readonly parameters?: { [parameterName: string]: any }
+  readonly parameters: { [parameterName: string]: any };
```

Comment:
Is this a breaking change? If the parameter is no longer optional, does that mean we should remove the `@default` docstring?

Reply:
Yes, it would normally be a breaking change. However, this interface is not exported from this package, so  it's acceptable here.

Absolutely, the `@defailt` should be removed, good catch. Done.